test/OpenTelemetry.Tests/Logs/LogRecordTest.cs

```diff
@@ -107,6 +107,74 @@ public void CheckStateForUnstructuredLogWithStringInterpolation()
             Assert.Equal(message.ToString(), state.ToString());
         }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1602:Enumeration items should be documented", Justification = "enum for test")]
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.OrderingRules", "SA1201:Elements should appear in the correct order", Justification = "enum for test")]
```

Comment:
Is it possible to avoid these? (e.g. if we make this enum internal/private would it help?)

Reply:
yes, making it private did help with SA1602 and I fixed the ordering!